### PR TITLE
css-3-layout. -- Joshua Preciado

### PR DESCRIPTION
**Pull request by joshuapreciado**
PICC

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+          <h4>In-depth Onboarding</h4>
+        </div>
+      </div>
+      <div class="row">
+        <div class="column3">
+          <p>A QMS that includes modules for all<br>
+            stages of the product life cycle from<br>    Design Control to CAPA</p>
+        </div>
+        <div class="column3">
+          <p>Enzyme QMS integrates with your<br>
```
Comment (thebearingedge) [inline css-three-column-layout/index.html]: Try reorganizing this into 1 row of 3 columns, each column should have an image, heading, and paragraph in it.

File: css-three-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  position: absolute;
+  height: 15px;
+  padding-top: 30px;
+  padding-left: 388px;
+  padding-right: 80px;
+}
+
+.pictures3 {
+  display: block;
+  position: absolute;
```
Comment (thebearingedge) [inline css-three-column-layout/styles.css]: You won't need absolute positioning for this challenge.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by joshuapreciado** [comment]
image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**